ios/FluentUI/People Picker/Avatar.swift

```diff
@@ -28,6 +29,9 @@ open class AvatarData: NSObject, Avatar {
     public var secondaryText: String
     public var image: UIImage?
 
+    /// An image that can be used as a frame (outer wide broder) for the avatar view
```

Comment:
nit spellcheck: border?

Reply:
isn't there a caveat? the avatar view itself should also have has border to be true?

Reply:
I was thinking it was a good assumption that if a customBorderImage is set, the client intends the view to have a border (regardless of the hasBorder value). Also, hasBorder refers to the usual border, not the customBorder, I can make that clearer in documentation